db.js

```diff
@@ -0,0 +1,91 @@
+"use strict";
+
+const mongo = require('mongodb').MongoClient;
+const logger = require("@dojot/dojot-module-logger").logger;
+const config = require('./config');
+
+// Errors ...
+class DatabaseNotFound extends Error {
+    constructor(...args) {
+        super(...args);
+    }
+}
+
+class InternalError extends Error {
+    constructor(...args) {
+        super(...args);
+    }
+}
+// ... Errors
+
+class DB {
+    constructor() {
+        this.client = null;
+        this.databases = new Map();
+    }
+
+    init() {
+        return mongo.connect(config.cronManager.db.mongodb.url, 
+            config.cronManager.db.mongodb.options).then((client) => {
+            this.client = client;
+          });
+    }
+
+    setTenant(tenant) {
```

Comment:
This name does not sound clear in the class context

Reply:
Any suggestion?

Reply:
createDatabase(tenant)?